edgy/core/db/fields/factories.py

```diff
@@ -25,21 +54,25 @@ class FieldFactory(metaclass=FieldFactoryMeta):
 
     _bases: Sequence[Any] = (Field,)
```

Comment:
lets call it `__bases__` instead? Cosmetic reasons only since we override it in the examples